mods/ts/rules/gdi-support.yaml

```diff
@@ -101,27 +101,27 @@ GACTWR:
 		UpgradeTypes: tower.vulcan
 		UpgradeMinEnabledLevel: 1
 		Weapon: VulcanTower
-		LocalOffset: 768,85,512
+		LocalOffset: 416,85,960
 		MuzzleSequence: muzzle
 		MuzzleSplitFacings: 8
 	Armament@VULCSECONDARY:
 		UpgradeTypes: tower.vulcan
 		UpgradeMinEnabledLevel: 1
 		Name: secondary
 		Weapon: VulcanTower
-		LocalOffset: 768,-85,512
+		LocalOffset: 416,-85,960
 		MuzzleSequence: muzzle
 		MuzzleSplitFacings: 8
 	Armament@ROCKET:
 		UpgradeTypes: tower.rocket
 		UpgradeMinEnabledLevel: 1
 		Weapon: RPGTower
-		LocalOffset: 512,-128,512
+		LocalOffset: 192,-65,1056
 	Armament@SAM:
 		UpgradeTypes: tower.sam
 		UpgradeMinEnabledLevel: 1
 		Weapon: RedEye2
-		LocalOffset: 512,0,512
+		LocalOffset: 192,0,1152
```

Comment:
This looks wrong ingame to me - it comes from the middle of the turret instead of the front.